README.md

```diff
@@ -32,10 +32,13 @@ Docs are available at http://127.0.0.1:8000/v1/docs.
 
 ### Sample Queries
 
+Sample queries written in [httpie](https://httpie.io/)
+
 - http GET http://127.0.0.1:8000/health
 - http GET http://127.0.0.1:8000/v1/variableById/data/42539
 - http GET http://127.0.0.1:8000/v1/variableById/metadata/42539
```

Comment:
not related to this PR, but would be good to have examples for the `v1/dataset` endpoints too

Reply:
Good point, added covid dataset and maddison metadata.